public/source/index.php

```diff
@@ -708,9 +679,8 @@
         <p>Upon validation, clients MUST check the <code>me</code> value from the <a href="#profile-url-response">profile URL response</a> or <a href="#access-token-response">access token response</a>, and take the following validation steps:</p>
 
         <ol>
-          <li>It MUST follow any permanent redirections from this URL to discover the canonical profile URL, in the same manner as <a href="#discovery-by-clients">initial profile URL discovery</a>.</li>
-          <li>It MUST verify that the canonical profile URL is on the same domain as the initially-entered profile URL.</li>
-          <li>It MUST verify that the canonical profile URL declares the same <code>authorization_endpoint</code> as the initially-entered profile URL.</li>
+          <li>It MAY check the value against any URLs encountered during the <a href="#discovery-by-clients">initial endpoint discovery</a>, either from a possible redirect chain or the final value. If found, it MAY then chose to skip the next step.</li>
```

Comment:
Explicitly explain what this gains and why it's safe. Maybe move after next point.

Reply:
I prefer "skip the next step" over "could have skipped the previous step", which is why I put it first. Would something like this be enough explanation?

```suggestion
          <li>It MAY check the value against any URLs encountered during the <a href="#discovery-by-clients">initial endpoint discovery</a>, either from a possible redirect chain or the final value. If found, it MAY then chose to skip the next step as no new information will be learned.</li>
```

Reply:
> … as it already knows following this URL will resolve to the same authorization endpoint"

?

Reply:
or "...will discover the same authorization endpoint", since resolve is ambigious.

Reply:
Lets postpone this waiting for @aaronpk's rewrite of this section. And get the actual removal of same domain requirement merged first 😊